### PR TITLE
 Fix missing env in docker context generator task

### PR DESCRIPTION
**Pull request by TadCordle**
Patch for 0.9.13.

This PR is only for approval purposes and shouldn't be merged with master. Only `9e658d2` needs to be reviewed, the rest is from the `v0.9.12-gradle` tag.

### CODE REVIEW

**Comment by TadCordle** [comment]
@chanseokoh Yeah, going to fix maven in a different branch.

File: jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java [hunk 1]
```diff
@@ -120,6 +122,16 @@ public void testGenerateDockerContext_errorOnWindowsAppRootWithDriveLetter() {
     }
   }
 
+  @Test
+  public void testGenerateDockerContext_env() throws IOException {
+    task.generateDockerContext();
+
+    Path dockerfile = projectRoot.getRoot().toPath().resolve("build/jib-docker-context/Dockerfile");
```
Comment (chanseokoh) [inline jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java]: I think `getDockerfileLine("ENV")` will work, as long as you care to get the first ENV line only.

Comment (TadCordle) [reply jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java]: I think that's only available in the integration tests?

Comment (chanseokoh) [reply jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java]: I see the method at the end of this file? https://github.com/GoogleContainerTools/jib/blob/master/jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/DockerContextTaskTest.java#L160

**Comment by TadCordle** [comment]
Closing and releasing: https://github.com/GoogleContainerTools/jib/pull/1150

**Pull request closed without merging**